packages/taquito/src/wallet/operation.ts

```diff
@@ -64,7 +66,8 @@ export class WalletOperation {
       return typeof x !== 'undefined';
     }),
     first(),
-    shareReplay({ bufferSize: 1, refCount: true })
+    publishReplay(1),
```

Comment:
I think publishReplay will be removed in version 8 of RxJS. https://rxjs.dev/api/operators/publishReplay#publishreplay

Reply:
It's prob not that big of a deal, but just something to keep in mind if we some day want to upgrade to RxJS 8